katas/content/complex_arithmetic/complex_addition/Verification.qs

```diff
@@ -0,0 +1,54 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;    
+    open Microsoft.Quantum.Convert; 
+
+    operation ComplexAddExp(x : Complex, y : Complex) : Complex {
```

Comment:
We usually name the reference solutions `<task name>_Reference` for uniformity

Reply:
Renamed reference solutions.